packages/terra-form-select/src/terra-dev-site/test/form-select/DefaultValueZero.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Select from '../../../Select';
+
+const DefaultValueZero = () => (
+<Select defaultValue={0} id="defaultValueZero" >
+    <Select.Option value={0} display='Display 1' />
+    <Select.Option value={1} display='Display 2' />
+    <Select.Option value={2} display='Display 3' />
```

Comment:
This test component might be easier to understand if the displays correspond to the value. Either "Zero" "One" "Two" or "0" "1" "2"

Reply:
Yeah, let's update the display props to one of these suggestions.